deprecated/alfresco-transformer-base/src/main/java/org/alfresco/transformer/config/MTLSConfig.java

```diff
@@ -120,11 +125,23 @@ private boolean isKeystoreConfigured()
     }
 
     private RestTemplate createRestTemplateWithSslContext(SSLContextBuilder sslContextBuilder) throws NoSuchAlgorithmException, KeyManagementException {
-        SSLContext sslContext = sslContextBuilder.build();
-        SSLConnectionSocketFactory sslContextFactory = hostNameVerificationDisabled ? new SSLConnectionSocketFactory(sslContext, NoopHostnameVerifier.INSTANCE)
-                : new SSLConnectionSocketFactory(sslContext);
+        final SSLConnectionSocketFactoryBuilder sslConnectionSocketFactoryBuilder =
+                SSLConnectionSocketFactoryBuilder.create()
+                        .setSslContext(sslContextBuilder.build())
+                        .setTlsVersions(TLS.V_1_2, TLS.V_1_3);
```

Comment:
We have such restriction implemented in Repo, so adding it here would make it consistent in my opinion.